app/src/conferences/models.py

```diff
@@ -286,41 +286,34 @@ class Meta:
     def _pays_for(self, option_name):
         return getattr(self, option_name)
 
-    def _price_for(self, option_name):
-        breakfast_price = self.zosia.price_accomodation_breakfast - self.zosia.price_accomodation
-        dinner_price = self.zosia.price_accomodation_dinner - self.zosia.price_accomodation
-        return {
-            'accomodation_day_1': self.zosia.price_accomodation,
-            'dinner_1': dinner_price,
-            'breakfast_2': breakfast_price,
-            'accomodation_day_2': self.zosia.price_accomodation,
-            'dinner_2': dinner_price,
-            'breakfast_3': breakfast_price,
-            'accomodation_day_3': self.zosia.price_accomodation,
-            'dinner_3': dinner_price,
-            'breakfast_4': breakfast_price
-        }[option_name]
+    def _price_for(self, chosen):
+        if not chosen["accomodation_day"] and not chosen["dinner"] and not chosen["breakfast"]:
```

Comment:
If we will add `_day` to both `dinner` and `breakfast` you will be able to remove it from here:
`accomodation_day`->`accomodation`